docs/upgrade.html

```diff
@@ -35,6 +35,8 @@
         Please refer to the <code>KafkaConsumer</code> Javadoc for a more in-depth explanation of this change.</li>
     <li>Multiple regular expressions separated by commas can be passed to MirrorMaker with the new Java consumer via the --whitelist option. This
         makes the behaviour consistent with MirrorMaker when used the old Scala consumer.</li>
+    <li>Zookeeper dependency was removed from Streams API. Now Streams API uses Admin Client API to manage internal topics. If the Kafka cluster is secured the Streams apps require to have proper security privilege to create new topics.</li>
```

Comment:
I am not sure it makes sense to say `Admin Client API` since it doesn't really exist. I think it would be better to say that the Kafka protocol is used to manage topics instead of modifying ZooKeeper. Also, more important than ZooKeeper dependency (which sounds like a library dependency) is the fact that network and write access to ZooKeeper is no longer required.